### DISCUSSION TITLE
How to install on Ubuntu for ARM?

### DISCUSSION THREAD

**Discussion by patrick-5546** [dandavison/delta#1284] [Q&A]
Is there a way I can install delta on Ubuntu for ARM? The releases only have amd64 `.deb` files.

**Comment by jaythomas** [comment]
Not through the package manager, but I was able to install it from cargo: `cargo install git-delta`. You need to add `$HOME/.cargo/bin` to your `$PATH` to run binaries installed from cargo though if you haven't already.
